sdk/core/Azure.Core.Expressions.DataFactory/src/DataFactoryKeyVaultSecret.Serialization.cs

```diff
@@ -9,7 +9,9 @@
 namespace Azure.Core.Expressions.DataFactory
 {
     [JsonConverter(typeof(DataFactoryKeyVaultSecretConverter))]
-    public partial class DataFactoryKeyVaultSecret : IUtf8JsonSerializable
+#pragma warning disable SCM0005 // Type must have a parameterless constructor
```

Comment:
Remove this warning suppression, it already has a parameterless constructor.
Apply this to other changes in this PR.